src/main/java/hudson/plugins/ec2/EC2Cloud.java

```diff
@@ -975,4 +971,32 @@ public static void log(Logger logger, Level level, TaskListener listener, String
         }
     }
 
+    @Extension
+    public static class EC2ConnectionUpdater extends PeriodicWork {
+        @Override
+        public long getRecurrencePeriod() {
+            return TimeUnit.SECONDS.toMillis(60);
+        }
+
+        @Override
+        protected void doRun() throws IOException {
+            Jenkins instance = Jenkins.get();
+            if (instance.clouds != null) {
+                for (Cloud cloud : instance.clouds) {
+                    if (cloud instanceof EC2Cloud) {
+                        EC2Cloud ec2_cloud = (EC2Cloud) cloud;
+                        LOGGER.finer("Checking EC2 Connection on: " + ec2_cloud.getDisplayName());
```

Comment:
```suggestion
                        LOGGER.finer(() -> "Checking EC2 Connection on: " + ec2_cloud.getDisplayName());
```

Lazy logging message construction for lower log levels.